Dockerfile

```diff
@@ -47,10 +44,15 @@ RUN \
   chmod 0700 /kms/extract-kms-str.py && \
   chmod 0700 /kms/convert-kms-private-ssh-key.sh && \
   chmod 0644 /etc/cron.d/ghe-backup && \
-  chmod 0700 /delete-instuck-backups/delete-instuck-progress.py && \
+  chmod 0700 /delete-instuck-backups/delete_instuck_progress.py && \
   mkfifo /var/log/ghe-prod-backup.log
 
-CMD python3 /delete-instuck-backups/delete-instuck-progress.py && \
+# delete_instuck_progress log
+RUN \
+  touch /var/log/ghe-delete-instuck-progress.log && \
+  chown -R application: /var/log/ghe-delete-instuck-progress.log
+
+CMD python3 /delete-instuck-backups/delete_instuck_progress.py && \
```

Comment:
I would advice to change this command to the format `CMD ["python3", "/delete-instuck-backups/delete_instuck_progress.py", ...]` to avoid surprises. ref: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/cmd